src/chapters/chapter1.tex

```diff
@@ -32,7 +32,7 @@ \section{Mathematical Notation}
 	It is tradition to use capital letters to name sets.  So we might say $A=\{6,7,12\}$
 	or $X=\{7\}$.  There is, however, a special set with a special name---the
 	empty set.  The \emph{empty set} is the set containing no elements
-	and is written $\emptyset$ or $\Set{}$.  Note that $\Set{\emptyset}$ is \emph{not}
+	and is written $\varnothing$ or $\Set{}$.  Note that $\Set{\varnothing}$ is \emph{not}
```

Comment:
`\emptyset` has more semantic meaning than `\varnothing`.  The correct way to do this change would be to redefine `\emptyset` to `\varnothing` in the preamble to the book.